client/cli/go/cmds/volume.go

```diff
@@ -383,3 +388,46 @@ var volumeListCommand = &cobra.Command{
 		return nil
 	},
 }
+
+var volumeCloneCommand = &cobra.Command{
+	Use:     "clone",
+	Short:   "Creates a clone",
+	Long:    "Creates a clone",
+	Example: "  $ heketi-cli volume clone 886a86a868711bef83001",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		//ensure proper number of args
+		s := cmd.Flags().Args()
+		if len(s) < 1 {
+			return errors.New("Volume id missing")
+		}
+
+		// Set volume id
+		volumeId := cmd.Flags().Arg(0)
```

Comment:
I would prefer not to have positional args. Refer to any other sub command of volume.

Reply:
@raghavendra-talur other sub-commands like `info` and `delete` also use positional args. Only the sub-commands that need more than a volume id seem to use named arguments. (This is apparently copy and pasted from `info`.)

Reply:
We discussed this a while ago, and I explicitly requested @nixpanic to do it like this (I think I even did the change to the CLI myself), because just as @ansiwen said, this is consistent with the other commands. And the main argument is that a CLI operation should (imho) always be a positional argument. Like the volume ID to clone, in this case.